woocommerce/woocommerce
# WP & PHP Upgrade Notices

### All Submissions:






### Changes proposed in this Pull Request:
This PR introduces a dashboard widget and a banner that notifies and prompts users to upgrade their WordPress and/or PHP if they are outdated and not part of the minimum required versions going into WooCommerce 3.7.

This PR needs to go out in the 3.6 release.

![Dashboard Widget](https://cld.wthms.co/vAIzri+)

![Banner](https://cld.wthms.co/fl8IT2+)


### How to test the changes in this Pull Request:

1. Checkout branch
2. Update the WC_MIN_PHP_VERSION and/or WC_MIN_WP_VERSION constants in class-woocommerce.php to versions higher than your current WP and PHP versions.
3. Check the dashboard widget loads and the banner is displaying
4. Test dismissing the banner and ensuring the widget also does not load anymore.

### Other information:




### Changelog entry

> New - WordPress & PHP upgrade nudges when running older versions.
